server/core/basic_cluster.go

```diff
@@ -268,6 +268,42 @@ func (bc *BasicCluster) GetAverageRegionSize() int64 {
 	return bc.Regions.GetAverageRegionSize()
 }
 
+// GetStoresLeaderWriteRate get total write rate of each store's leaders.
+func (bc *BasicCluster) GetStoresLeaderWriteRate() (storeIDs []uint64, bytesRates, keysRates []float64) {
+	bc.RLock()
```

Comment:
it could decrease lock time

Reply:
Getting `len(bc.Stores.stores)` requires a read lock.